### PR TITLE
Hook up celery to Sentry

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

**Comment by andymckay** [comment]
r+

**Comment by kumar303** [comment]
Travis won't cover any of these changes so I'll just cowboy it...

File: docker/supervisor-celery.conf [hunk 1]
```diff
@@ -2,7 +2,13 @@
 logfile=/code/logs/supervisord-celery.log
 
 [program:olympia]
-command=celery worker --app=olympia
+# This command mimics how we run it in prod from puppet:
+# https://github.com/mozilla-services/puppet-config/blob/master/amo/modules/olympia/manifests/celery.pp#L16-L24
```
Comment (magopian) [inline docker/supervisor-celery.conf]: you might want a commit hash in the url? So the link stays relevant even if the file changes?

File: olympia.py [hunk 2]
```diff
@@ -72,13 +80,33 @@ def init_amo():
...
+    client = Client(settings.SENTRY_DSN)
+
+    # register a custom filter to filter out duplicate logs
+    register_logger_signal(client)
+
+    # hook into the Celery error handler
+    register_signal(client)
+
+    # After upgrading raven we can specify loglevel=logging.INFO to override
+    # the default (which is ERROR).
```
Comment (magopian) [inline olympia.py]: This is a bit vague: upgrading to what version?

File: lib/settings_base.py [hunk 3]
```diff
@@ -1477,3 +1477,7 @@ def read_only_mode(env):
         'rest_framework.renderers.JSONRenderer',
     ),
 }
+
+# This is the DSN to the local Sentry service. It might be overidden in
+# site-specific settings files as well.
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
```
Comment (magopian) [inline lib/settings_base.py]: I don't think this is configured in docker yet, is it? Do we already have a local sentry service?

**Comment by magopian** [comment]
For reference, this "repairs" the connection between celery and sentry, which was most probably broken by the update in #678

**Pull request merged**